### DISCUSSION TITLE
8.0.6 Release - Aniversary edition

### DISCUSSION THREAD

**Discussion by Donnnno** [Arcticons-Team/Arcticons#1781] [Announcements]
Hey everyone, around this time, 3 years ago, I released the first version of Arcticons with around 900~ icons, now we have 8000+!

After contributing for months to [Frost](https://github.com/dkanada/frost), a fill-based icon pack. I wanted to create something different (but with the same goal), a line-based icon pack. But I lacked the technical skills to create an app, gladly I came across Frost, which was a perfect app to use as a base for Arcticons.

In these three years, the project gained many contributors, and thanks to its low entry level. We’ve managed to do quite a lot in that time, like: Upgrading our app to have more features like icon request support, a variant with [black lines](https://f-droid.org/en/packages/com.donnnno.arcticons.light/) and later one with [Material You colors](https://f-droid.org/packages/com.donnnno.arcticons.you/), we’ve also created 7000+ icons together, and we’re expanding to Linux too!

This is all possible thanks to the open-source nature of the project, and it’s fantastic community. :snowflake:

# 8.0.6 - Anniversary update
###### Released: Nov 8, 2023
* 🎈 Arcticons turned 3 years old, thanks everyone, for being with us, making this possible!
* 🎉 550 new and updated icons! (thanks @Onuronsekiz, @Petlyh, @Kaiserdragon2, @Starfetch, @MikeTheGreat, @MrAlpha786)
* ✨ Updates to make sure many apps work again.
* 🔥 8068 icons in total!

https://github.com/Donnnno/Arcticons/assets/31142286/ee706a73-cfeb-4464-bd61-be10666faeb7

![Untitled-1](https://github.com/Donnnno/Arcticons/assets/31142286/941b3226-1a0a-4362-9f13-96d7912d903c)
![Untitled-12](https://github.com/Donnnno/Arcticons/assets/31142286/d8a90496-3140-4be9-9c3d-9af98529cfff)

<hr /><em>This discussion was created from the release <a href='https://github.com/Donnnno/Arcticons/releases/tag/8.0.6'>8.0.6 Release - Aniversary edition </a>.</em>

**Comment by xprateek** [comment]
### Amazing...🎉
